about/release-notes/index.md

```diff
@@ -4,14 +4,17 @@ excerpt: New features and fixes are released regularly
 keywords: [upgrades, updates, releases]
 ---
 
-# TimescaleDB release notes
+# Timescale release notes
 
-TimescaleDB is an open-source project with a vibrant community. To get involved,
-join our [community][community].
+This page contains release notes for Timescale&nbsp;2.10.0 and newer. For
+release notes for older versions, see the
+[past releases section][past-relnotes].
 
 <Highlight type="note">
-Want to stay up-to-date with new releases? Subscribe to get
-[release notes email updates](https://www.timescale.com/signup/release-notes).
+Want to stay up-to-date with new releases? You can subscribe to new releases on
+GitHub and be notified by email whenever a new release is available. On the
+[Github page](https://github.com/timescale/timescaledb),
```

Comment:
I am not sure if this will be OK in the final version, but when checking the md directly, the link inside the highlight is rendered as is, not as a link

https://github.com/timescale/docs/blob/e1f117dbca49fafb5ea9f33dc66be9ed2d87f559/about/release-notes/index.md

Reply:
Unfortunately, I screwed up creating the preview properly. But once we've merged this into the launch branch, we'll be able to see it and make sure it's working correctly. That branch will also have a bunch of link checking and redirects before it merges anyway 👍